search-resources-api/src/test/resources/roundTripImportCandidateResponse.json

```diff
@@ -0,0 +1,74 @@
+{
+  "@context": "https://api.nva.unit.no/resources/search",
+  "id": "https://localhost/search?query=searchTerm",
+  "processingTime": 49,
+  "size": 2,
+  "hits": [
+    {
+      "type": "ImportCandidate",
```

Comment:
This is not the search model we are going to use.

Reply:
See task: https://unit.atlassian.net/browse/NP-44692

Reply:
Waiting til object is implemented